pa11y/serve.py

```diff
@@ -0,0 +1,66 @@
+""" a minimal static server, primarily for performing audits
+
+    this is not meant to be used in production, and some settings are insecure
```

Comment:
```suggestion
    DO NOT USE IN PRODUCTION. Some settings are insecure
```

Reply:
made it even more shoutier than proposed, and explained some of the deltas